src/monitor/check_restart.py

```diff
@@ -0,0 +1,56 @@
+import time
+import subprocess
+from pathlib import Path
+from watchdog.observers import Observer
+from watchdog.events import FileSystemEventHandler
+
+PROJECT_DIR = Path(__file__).resolve().parents[2]
```

Comment:
Add strict variables to become global variables to keep it easy to see.
```suggestion
PROJECT_DIR = Path(__file__).resolve().parents[2]
MONITOR_EXEC = PROJECT_DIR / 'jobs/monitor_exec'
```